### PR TITLE
add debate workbench to meinberlin

### PR DESCRIPTION
**Pull request by Johnschmidfront**

### CODE REVIEW

**Comment by Johnschmidfront** [comment]
@joka @pallix please review backend. thx!

**Comment by xi** [comment]
frontend looks fine

File: src/adhocracy_meinberlin/adhocracy_meinberlin/resources/test_collaborative_text.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    def test_create(self, registry, meta):
+        assert registry.content.create(meta.iresource.__identifier__)
+
+def _post_document_item(app_user, path='') -> TestResponse:
+    from adhocracy_core.resources.document import IDocument
+    resp = app_user.post_resource(path, IDocument, {})
+    return resp
+
+
+@mark.functional
```
Comment (pallix) [inline src/adhocracy_meinberlin/adhocracy_meinberlin/resources/test_collaborative_text.py]: can you move this test to the 'workflows' folder? Sorry for the inconvenience but we decided only yesterday that's it's more coherent.

**Comment by Johnschmidfront** [comment]
I haven't tested this yet, so don't merge please

**Comment by pallix** [comment]
Only the functional (with 'mark.functional') tests should be moved to workflows. JSON files should be removed from `resources`

do "polytester pyfunc, pyunit" after and checks the the coverage ist by 100% in `htmlcov/index.html`

**Comment by Johnschmidfront** [comment]
works.

**Pull request merged**